scripts/publish-process/update-docs-deps.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Backpack - Skyscanner's Design System
+#
+# Copyright 2016-2021 Skyscanner Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BRANCH_NAME="autobump-upgrade-bpk-dependencies"
+
+echo "--- Cloning Backpack docs ---"
+git clone git@github.com:skyscanner/backpack-docs.git
+
+cd backpack-docs
+
+echo "--- Checkout upgrade branch ---"
+git checkout -b $BRANCH_NAME
```

Comment:
This will fail if the branch already exists remotely. Maybe it should be this + timestamp?

Reply:
Only the case if you doon't delete the branch after merging the PRs ;)

Reply:
Which could easily happen, especially as we sometimes release a lot

Reply:
Its also worth noting we do have this setting turned on in the docs repo meaning that the branch should never exist on remote once the PR has been merged 🤷 

![Screenshot 2021-03-25 at 13 38 27](https://user-images.githubusercontent.com/8831547/112481910-66c4ec80-8d6f-11eb-93da-2d97268d6bd6.png)

Reply:
Yeah but suppose I release something small, I might not look at the docs at all, and then someone releases something later and suddenly the PR they actually want isn't made because the branch already existed

Reply:
That is true, I just went on the assumption our PRs channel would alert that a PR has been opened and we would be jumping on it asap as normal, but I do see this as a potential clash!

Reply:
Changed this to use the date and time at the end to avoid the problem @georgegillams has mentioned above